LanternMobileTestbed/app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,12 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <service
+            android:name="org.lantern.mobilesdk.service.LanternService"
+            android:exported="false"
+            android:process=":LanternService"
+            />
```

Comment:
Good questions.

1. Making the service global seems like a good idea

2. If you're running the Lantern VPN app and an app the embeds Lantern simultaneously, I imagine that the embedded Lantern will proxy via the VPN, so we'll be doing double-duty on both the client and server-side.